apollo-federation/src/sources/connect/models/validation.rs

```diff
@@ -321,7 +384,7 @@ mod test_validate_source {
         let schema = Schema::parse(schema, "test.graphql").unwrap();
         let errors = validate(schema);
         assert_eq!(errors.len(), 1);
-        assert!(matches!(&errors[0], ValidationError::SourceScheme { .. }));
+        assert_snapshot!(errors[0].to_string());
```

Comment:
for short-ish string assertions, can you use [inline snapshots](https://docs.rs/insta/latest/insta/#inline-snapshots) so it's easier to see the expected result in the test? (or just use `assert_eq`)

Reply:
Seems like the sort of thing we should maybe have an _overall_ style guide for—so the errors are consistent across products. Is there already such a thing somewhere or should we kick that off here and with the consuming work.